project/Build.scala

```diff
@@ -982,6 +982,7 @@ object Build {
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/lang/ref" ** ("ReferenceTest.scala": FileFilter)).get
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/lang/reflect" ** ("ReflectArrayTest.scala": FileFilter)).get
           ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/io" ** (("ThrowablesTest.scala": FileFilter) || "SerializableTest.scala" || "PrintWriterTest.scala")).get
+          ++ (dir / "shared/src/test/scala/org/scalajs/testsuite/javalib/math" ** (("BigDecimalTest.scala": FileFilter) || "RoundingModeTest.scala" || "MathContextTest.scala" || "BigIntegerXorTest.scala" || "BigIntegerToStringTest.scala" || "BigIntegerTest.scala" || "BigIntegerSubtractTest.scala" || "BigIntegerOrTest.scala" || "BigIntegerOperateBitsTest.scala" || "BigIntegerNotTest.scala" || "BigIntegerMultiplyTest.scala" || "BigIntegerModPowTest.scala" || "BigIntegerHashCodeTest.scala" || "BigIntegerDivideTest.scala" || "BigIntegerConvertTest.scala" || "BigIntegerConstructorsTest.scala" || "BigIntegerCompareTest.scala" || "BigIntegerAndTest.scala" || "BigIntegerAddTest.scala" || "BigDecimalScaleOperationsTest.scala" || "BigDecimalConvertTest.scala" || "BigDecimalConstructorsTest.scala" || "BigDecimalCompareTest.scala" || "BigDecimalArithmeticTest.scala")).get
```

Comment:
Could you use a single `"*.scala"` instead of listing all the tests, please?

Reply:
Sure